wormhole-connect/src/utils/style.ts

```diff
@@ -87,3 +88,29 @@ export const changeOpacity = (
   if (color.length === 9) return color.slice(0, 7) + OPACITY[opacity];
   return color;
 };
+
+const useScrollbarClassName = makeStyles()((theme) => ({
```

Comment:
Can you add a comment here to explain why we need this hook to generate the scrollbar styles?